docs/administrators/logging.rst

```diff
@@ -12,69 +12,46 @@ configured regarding to logging.
+2. ``resource-actions-<environment-id>.log`` contains the all actions performed by all resources. Each environment has one resource action log file. This file mirrors the actions logged on resources in the database.
+3. ``agent-<environment-id>.log`` is the main log of the scheduler. It contains information about when any executor started a deployment, which trigger caused that deployment, whether heartbeat messages are received from the server, etc.
+4. ``agent-<environment-id>.out``: This log file contains all the messages written to the standard output stream of the scheduler and executors. Expected to be empty.
+5. ``agent-<environment-id>.err``: This log file contains all the messages written to the standard error stream of the scheduler and executors. Expected to be empty.
 
-The ``server.log`` file contains general debugging information regarding the Inmanta server. It shows information about actions
-performed by the Inmanta server (renewing parameters, purging resource action logs, etc.), API requests received by the
-Inmanta server, etc.
+For reasons of backward compatibility, th scheduler files are called 'agent' and not 'scheduler'
 
 
-Resource action logs
---------------------
-
-The resource action log files contain information about actions performed on a specific resource. Each environment has one
-resource action log file. The filename of this log file looks as follows:
-``<server.resource-action-log-prefix>-<environment-id>.log``. The prefix can be configured with the configuration option
-:inmanta.config:option:`server.resource-action-log-prefix`.
-
-The resource action log file contains information about the following resource action:
-
-* **Store**: A new version of a configuration model and its resources has been pushed to the Inmanta server.
-* **Pull**: An agent pulled its resources from the Inmanta server.
-* **Deploy**: When an agent starts and ends the deployment of a certain resource.
-* **Dryrun**: Execute a dryrun for a certain resource.
-
-
-Agent logs
-----------
+Configure logging
+=================
 
-One agent produces the following three log files:
+Logging can be configured in two main way:
 
-* ``agent-<environment-id>.log``: This is the main log file of an agent. It contains information about when the agent
-  started a deployment, which trigger caused that deployment, whether heartbeat messages are received from the server,
-  whether the agent is a primary agent, etc.
-* ``agent-<environment-id>.out``: This log file contains all the messages written to the standard output stream of the resource
-  handlers used by the agent.
-* ``agent-<environment-id>.err``: This log file contains all the messages written to the standard error stream of the resource
-  handlers used by the agent.
+- course grained configuration using configuration and command line options. This is sufficient in most cases.
```

Comment:
```suggestion
- coarse grained configuration using configuration and command line options. This is sufficient in most cases.
```
Fancy word, never heard of it before, maybe could be turned into something more mainstream for more people? like "general configuration", "base configuration", etc